b9lab-content/3-main-concepts/02-architecture.md

```diff
@@ -1 +1,294 @@
+---
+title: "A Blockchain App Architecture"
+order: 2
+description: Architecture of a blockchain application built with Cosmos.
+---
+
 # Blockchain App Architecture
+
+## What is Tendermint
+
+Created in 2014, Tendermint accelerates the development of distinct blockchains by attending to general concerns so that those concerns don’t need to be recreated by developers in each case. You may already be using Tendermint without awareness of this since blockchains including [Hyperledger Burrow](https://hyperledger.github.io/burrow/#/) and [Binance Chain](https://www.binance.org/en/smartChain)  use Tendermint.
```

Comment:
```suggestion
Created in 2014, Tendermint accelerates the development of distinct blockchains by providing a ready-made networking and consensus solution so that these don’t have to be recreated by developers in each case. You may already be using Tendermint without awareness of this since blockchains including [Hyperledger Burrow](https://hyperledger.github.io/burrow/#/) and [Binance Chain](https://www.binance.org/en/smartChain)  use Tendermint.
```
Vague use of “concerns”.